### DISCUSSION TITLE
Есть ли возможно смещать центр карты, если балун отображается у края карты

### DISCUSSION THREAD

**Discussion by Denisoov** [yandex-maps-unofficial/vue-yandex-maps#286] [Q&A (2.x)]
При клике на край карты, по координатам устанавливается маркер аля балун с информацией. Но центр карты не смещается по координатам, чтобы отображение маркера было во viewport яндекс карты 
![image](https://github.com/yandex-maps-unofficial/vue-yandex-maps/assets/48566682/e5bd1da6-1e19-42ef-b00a-fc5ba791cdd0)

Можно ли реализовать такой кейс ?

**Comment by Bogrado** [comment]
Удалось решить?

**Comment by mihaon** [comment]
Думаю можно вычислять выходит ли балун за пределы viewport'а карты и сдвигать его. Пример вычисления положения маркера относительно viewport'а карты в процентах здесь - https://github.com/yandex-maps-unofficial/vue-yandex-maps/issues/219#issuecomment-2708494713
